config/squidGuard/squidguard_configurator.inc

```diff
@@ -91,11 +91,20 @@ define('CONFIG_SG_HEADER', "
 # ------------------------------------------------------------------------------
 # squid config options
 # ------------------------------------------------------------------------------
+
+$pf_version=substr(trim(file_get_contents("/etc/version")),0,3);
+
 define('REDIRECTOR_OPTIONS_REM',   '# squidGuard options');
-define('REDIRECTOR_PROGRAM_OPT',   'redirect_program');
-define('REDIRECT_BYPASS_OPT',      'redirector_bypass');
 define('REDIRECT_CHILDREN_OPT',    'url_rewrite_children');
-define('REDIRECTOR_PROCESS_COUNT', '5'); # redirector processes count will started
+if ($pf_version >= 2.2) { 
+   define('REDIRECTOR_PROGRAM_OPT',   'url_rewrite_program');
+   define('REDIRECT_BYPASS_OPT',      'url_rewrite_bypass');
+   define('REDIRECTOR_PROCESS_COUNT', '16 startup=8 idle=4 concurrency=0');# redirector processes count will started
```

Comment:
Does PHP cope OK with the end of command semi-colon directly followed by comment char ";#" ?
It would at least look nicer to have a space in there.
The comment came from the previous code, and must have been written by a non-first-language English speaker because actually I do not know what it means - kind of useless having a comment that is not understood. But that is not your fault.
Since there are more parameters here for the 2.2 case, the comment could be removed or re-written to explain what the 4 numbers are about.